docs/start-stop.rst

```diff
@@ -40,16 +40,15 @@ for further details on customizing the UnifyFS runtime configuration.
 .. code-block:: Bash
     :linenos:
 
-        #!/bin/bash
+    #!/bin/bash
 
-        # spillover checkpoint data to node-local ssd storage
-        export UNIFYFS_SPILLOVER_DATA_DIR=/mnt/ssd/$USER/data
-        export UNIFYFS_SPILLOVER_META_DIR=/mnt/ssd/$USER/meta
+    # spillover data to node-local ssd storage
+    export UNIFYFS_LOGIO_SPILL_DIR=/mnt/ssd/$USER/data
 
-        # store server logs in job-specific scratch area
-        export UNIFYFS_LOG_DIR=$JOBSCRATCH/logs
+    # store server logs in job-specific scratch area
+    export UNIFYFS_LOG_DIR=$JOBSCRATCH/logs
 
-        unifyfs start --share-dir=/path/to/shared/file/system &
+    unifyfs start --share-dir=/path/to/shared/file/system &
```

Comment:
Based on Hyogi's recent work, I suggest removing ampersand at the end of this line. There's no need to background the process anymore.

Reply:
Ah, yup. Nice catch! I'll update that.